deploy/cloudformation/static-host-pipeline.yml

```diff
@@ -14,6 +14,14 @@ Parameters:
     Type: String
     Description: An e-mail address of someone who can approve test environment
 
+  ApiURL:
+    Type: String
+    Description: URL of API without trailing slash.
+
+  TargetURL:
```

Comment:
Would be better if we could disambiguate this name a bit. It's unclear what "target" means here without digging in to find out that it means the URL that will be posted to the github status.

Reply:
Actually, you could even just reuse the ProdURL param here too, since this is what the pipeline is deploying to